roles/manage-ec2-instances/tasks/provision.yml

```diff
@@ -237,7 +237,8 @@
       Info: "Username that provisioned this-> {{ skylight_user }}"
       Skylight: "This was provisioned through the skylight provisioner"
       Students: "{{ user_count }}"
-      short_name: 'windc'
+      short_name: "{{ 's' + item + '-windc' }}"
```

Comment:
I believe this was a mistake, there is only 1 DC and it shouldn't prefixed with a student #

Reply:
Well, it wasn't done in mistake; I was trying to build out a sane DNS structure for public and didn't realize there was a technical reason.  perhaps I should only mess with long_name?

Reply:
Actually, I don't think it would break anything, as we don't use the short_name anywhere else important and we are hard coding the name as windc when we add_host.  Ideally it still shouldn't have a student number at the beginning though, as its misleading.